DRN.Framework.Hosting/README.md

```
@@ -415,39 +415,183 @@ protected override void ConfigureSecurityHeaderPolicyBuilder(
+
+- Rules execute in ascending `Order`. Framework defaults use `int.MaxValue`. Your rules run first.
+- Multiple matching rules **compose** via .NET's native chained limiter (e.g., tenant + user + IP all enforce together).
+- `ShortCircuitOnMatch = true`. Rule runs before normal same-order rules. A non-null match stops remaining rules. Use for allow/deny rules that must take precedence.
+- Use a lower `Order` when an allow/deny rule must bypass earlier quotas entirely.
+
+#### Partition Key Isolation
 
-protected override void ConfigureApplicationPostAuthorization(WebApplication application)
+DRN rate limit rules namespace every partition key with the phase and the rule type full name:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)